andela/vidar-ah-backend
# #164139686 Creating an article

#### What does this PR do?
This PR implement the functionality of creating an article by a user.

#### Description of Task to be completed?
- Setup an endpoint to post articles.
- Ensures that all required inputs are sent.
- Provides swagger documentation for this task.

#### How should this be manually tested?
- Run `npm run start:dev` to start the application.
- Check .env.sample for the updated .env variables.
- Run `npm run start` to start the application.
- Make a POST request to `/api/v1/articles` with a JSON object containing the following fields: `title`, `description`, and `body`. They're all required. If one is not sent, there'll be an error.
- If this request is successful, you should receive a success message.

#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
#164139686
